### ISSUE TITLE
Proposal: No more KitchenSink

### ISSUE THREAD

**Issue by blesh** [ReactiveX/rxjs#1650]
We should probably just have one global export file for the masses.

1. Is it huge? Yes.
2. Should people use RxJS 5 that way? Hell no. Use a bundler and only import what you're using.
3. In past tests, I noticed the gzipped size of Rx.KitchenSink.min.js was about the same as Rx.min.js, because of how the compression affected it.

If people are using the global export version of RxJS, it can be assumed that they're probably hacking around, and/or not really serious about reducing the size of their downloaded application.

At this point it's a little confusing to say "this is in Rx, but this is in Kitchensink" to people. Also, given how modular this library is, it would be easy enough for anyone to make their own custom build of RxJS 5 to keep around.

### Basically the idea is just to put everything in Rx.ts

Thoughts?

**Comment by kwonoj** [comment]
I'm in favor of this in general too, but had assumption those packages are similar representation of `rx.all.js` to `rx.lite.js` in RxJS4. So in case of anyone who'd like to have minimal, core packages (`Rx` currently), it's recommended to make custom imported builds of their own?

**Comment by staltz** [comment]
Agreed.

**Comment by trxcllnt** [comment]
Been in favor of this for a long time, let's do it.

**Comment by blesh** [comment]
> So in case of anyone who'd like to have minimal, core packages (Rx currently), it's recommended to make custom imported builds of their own?

Yes. If someone cares that much they should be using bundling and tree shaking anyhow.

**Comment by zertosh** [comment]
I would love to see this go out in a beta 8.
